health_check/contrib/beat_health_check/backends.py

```diff
@@ -0,0 +1,100 @@
+from datetime import timedelta
+from typing import Any, Dict, List, Union
+
+from celery.beat import Service
+from celery.schedules import crontab, solar, schedule
+from django.conf import settings
+from django.utils.module_loading import import_string
+from django_celery_beat.models import CrontabSchedule, IntervalSchedule, SolarSchedule
+from django_celery_beat.schedulers import ModelEntry
+from health_check.backends import BaseHealthCheckBackend
+from health_check.exceptions import ServiceReturnedUnexpectedResult, ServiceUnavailable
+
+
+class CeleryBeatHealthCheck(BaseHealthCheckBackend):
+    def check_status(self):
+        """
+        Checks for overdue tasks in a celery beat scheduler.
```

Comment:
The first-line in docstrings should be short, general sentence about the function/method, etc. Then empty line and then longer, more detailed description